ENIGMAsystem/SHELL/Widget_Systems/None/nowidget_impl.cpp

```diff
@@ -25,6 +25,49 @@ using std::string;
 
 #include "GameSettings.h"
 
+#include <iostream>
```

Comment:
This change is sufficiently complex to warrant updating the copyright headers. Just extend the years to today under both our names.

Reply:
Agreed. d18de1aa95556fe70e01025d87b15770e2db48e2